odoo/http.py

```diff
@@ -64,6 +64,8 @@
 # To remove when corrected in Babel
 babel.core.LOCALE_ALIASES['nb'] = 'nb_NO'
 
+ALLOWED_DEBUG_MODES = [False, None, '', '1', 'assets', 'tests']
```

Comment:
It would be kind of weird to allow passing debug=1 to enable debug, but not debug=0 to disable it, don't you think? I imagine `'0'` could be aliased to `False` by `_dispatch()`, and `'1'` aliased to True?
Not sure about `None`, is it actually used with the current code paths? If unnecessary just omit it, so it's not tested every time.

Reply:
Indeed, I used [str2bool](https://github.com/odoo-dev/odoo/blame/b3c6f95645d0c487000546c5ef3eb328e09da841/odoo/tools/misc.py#L440) and it will now manage such cases logically.
That allowed also to clean `ALLOWED_DEBUG_MODES` to only [False, True, 'assets', 'tests']